tests/clients/test_detectionlists.py

```diff
@@ -113,11 +113,10 @@ def test_create_user_when_service_returns_cannot_find_user_bad_request_raises_un
             client.create_user("testusername")
 
         assert (
-            str(err.value)
-            == "Detection-list profiles are now created automatically on the server. "
+            "Detection-list profiles are now created automatically on the server. "
             "Unable to find a detection-list profile for 'testusername'. It is "
             "possibly still being created if you just recently created the Code42 "
-            "user."
+            "user." in str(err.value)
         )
```

Comment:
Can we also add some tests that the new properties are what we expect? Like: 

```
assert err.value.username == 'testusername'
```

Reply:
oh yeah thats a good point, for sure!